src/stylesheets/components/_navigation.scss

```diff
@@ -419,7 +419,7 @@
     flex-direction: column;
 
     @include media($nav-width) {
-      display: initial;
+      display: block;
```

Comment:
Er wait, I'm being silly. I think just replacing `initial` with `block` is fine!